### ISSUE TITLE
Darcula Scheme support (dark background, light font)

### ISSUE THREAD

**Issue by rosivda** [ShyykoSerhiy/gfm-plugin#73]
For programmers who configured their IntelliJ for Darcula Scheme (dark background, light font), the "Text" mode appears normal but the "GFM Preview" retains white background and dark font. This goes contrary to the Darcula Scheme. Thus the following request:

For Darcula Scheme, the font and background color should be reversed (background dark, font light). In the fix prototype submitted by Serhiy Shyyko on July 28, 2015 @ 5:15 AM, the appearance of the GFM Preview is exactly what is been requested here. Thanks for addressing this.

**Comment by ShyykoSerhiy** [comment]
connected issue #54

**Comment by ShyykoSerhiy** [comment]
Will be added as a part of #54. 
Easiest way will be add 
```css
html{
       background-color: #333333;
}
body{
       -webkit-filter: invert(80%);
}
img{
       -webkit-filter: invert(100%);
} 
```
as additionalCss param in settings.
![additionalcss](https://cloud.githubusercontent.com/assets/1106995/8944101/2bb0c268-3587-11e5-8b34-b5a802f2a27b.gif)

**Comment by rosivda** [comment]
This additional css styling option would indeed be a convenient solution for us to use. Thanks.

**Comment by ShyykoSerhiy** [comment]
fixed in v0.1.5
